### DISCUSSION TITLE
Is there a better way: Page links

### DISCUSSION THREAD

**Discussion by Linwood-F** [Blackymas/NSPanel_HA_Blueprint#1279] [Q&A] [answered]
At present on the home page I have an icon linked to a script, e.g. 

    front_panel_page_display: 
      sequence: 
        - service: esphome.frontdoorpanel_send_command_printf
          data:
            cmd: page buttonpage02

so to jump from home to page 2 they can touch that icon. 

This works, but seems a bit round-about, with the round trip to HA and an automation and command.

Is there any way to link from home page to other pages directly, or is this the right approach?

**Comment by edwardtfn** [comment, marked as answer]
You can swipe left, right, up and down, each will move you to one of the 4 different button pages.
Is this what you are looking for?

**Comment by Linwood-F** [comment]
I realize that, was looking for a directed way, off a button.  But yes, it's only two swipes vs one button.

Mostly I was wondering if there was just a direct way within the unit, as opposed to the path back and forth to HA; curiosity whether I missed something.

**Comment by Doughboy68** [comment]
Maybe having navigation buttons in addition to swiping may be more intuitive. Also some settings from the home screen eg. climate and thermostat (pressing temperature) are not very intuitive and a new user may not know to press them. My wife figured out how to swipe screens and even long pressing one of the buttons brings up appropriate screen (ie. light, fan) but I don't think she figured out how to do climate/thermostat (maybe is good thing LOL). Wonder how to make those touch areas stand out without making it look ugly?
